app/services/licences/fetch-customer-contacts.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+/**
+ * Fetches all return logs for a licence which is needed for the view '/licences/{id}/contact-details` page
```

Comment:
NIT - copy and paste correction

```suggestion
 * Fetches all customer contacts for a licence which is needed for the view '/licences/{id}/contact-details` page
```